index.template.html

```diff
@@ -1139,6 +1166,7 @@
                 <div class="gen-specific g3 g4 g5 g6 g7">
                     <label for="abilityR1">Ability</label>
                     <select class="ability terrain-trigger calc-trigger" id="abilityR1"></select>
+                    <input hidden type="checkbox" class="abilityToggle calc-trigger" checked>
```

Comment:
We should probably have a title to explain what that checkbox is doing

Reply:
yeah good idea